src/backlight/positions/positions.py

```diff
@@ -83,7 +86,11 @@ def calc_positions(
     pos = Positions(_pricer(trades, mkt, principal))
     pos.reset_cols()
     pos.symbol = trades.symbol
-    pos.currency_unit = trades.currency_unit
+    if trades.currency_unit == None:
+        pos.currency_unit = Currency[pos.symbol[-3:]]
```

Comment:
Use `pos.currency_unit`?

Reply:
These problems have been solved in the previous PR.